libsrc/osic1p/bootstrap.s

```diff
@@ -45,6 +45,8 @@ GETCHAR         :=      $FFBF           ; gets one character from ACIA
 FIRSTVISC       =       $85             ; Offset of first visible character in video RAM
 LINEDIST        =       $20             ; Offset in video RAM between two lines
 
+        lda     #0
+        sta     load
```

Comment:
Thanks for the explanations. Will you fix this in your repository and submit the fix to the main repository, or should I do that from my repository?